src/eos-shard-writer-v2.c

```diff
@@ -328,7 +331,7 @@ write_blob (EosShardWriterV2 *self, struct eos_shard_writer_v2_blob_entry *blob,
     offset += size;
   }
 
-  close (blob_fd);
+  g_assert (close (blob_fd) == 0 || errno == EINTR);
```

Comment:
I thought `G_DISABLE_ASSERT` was basically assumed to be never defined because of side effects in asserts / `g_return_if_fail`s...